libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.scss

```diff
@@ -5,9 +5,11 @@
   display: flex;
 }
 
-.sky-date-range-picker-form-group {
-  &:last-child {
-    margin-bottom: 0;
+@media (max-width: $sky-screen-sm-min) {
+  .sky-date-range-picker-form-group {
+    &:not(.sky-date-range-picker-last-input) {
+      margin-bottom: $sky-space-md;
```

Comment:
if you use `var(--sky-margin-stacked-lg)` I think you won't have to specify different styles for modern theme